packages/mdc-drawer/mdc-drawer.scss

```diff
@@ -14,6 +14,4 @@
 // limitations under the License.
 //
 
-@import "./persistent/mdc-persistent-drawer";
-@import "./permanent/mdc-permanent-drawer";
-@import "./temporary/mdc-temporary-drawer";
+@import "./standard/mdc-drawer-standard";
```

Comment:
FYI for future PRs: We should figure out what common subset of styles applies across all drawers, and provide those in the context of `.mdc-drawer` here. We should avoid repeating the mistake of the current `mdc-drawer` which effectively repeated a bunch of the same styles for 3 separate selectors.

Reply:
Yup that makes sense. I think the next PR should give me a better idea of what can be in `.mdc-drawer`.